artemis-cli/src/main/java/org/apache/activemq/artemis/cli/commands/tools/DBOption.java

```diff
@@ -223,13 +225,17 @@ protected void initializeJournal(Configuration configuration) throws Exception {
       this.config = configuration;
       executor = Executors.newFixedThreadPool(5, ActiveMQThreadFactory.defaultThreadFactory());
       executorFactory = new OrderedExecutorFactory(executor);
-
-      scheduledExecutorService = new ScheduledThreadPoolExecutor(configuration.getScheduledThreadPoolMaxSize(), new ThreadFactory() {
+      scheduledExecutorService = new ScheduledThreadPoolExecutor(configuration.getScheduledThreadPoolMaxSize(), AccessController.doPrivileged(new PrivilegedAction<ThreadFactory>() {
          @Override
-         public Thread newThread(Runnable r) {
-            return new Thread(r);
+         public ThreadFactory run() {
```

Comment:
I know this inst your change, but it highlights a bigger question, as to why is this not using ActiveMQThreadFactory?